src/main/java/com/ably/kafka/connect/config/ConfigValueEvaluator.java

```diff
@@ -53,7 +53,7 @@ public Result evaluate(SinkRecord record, String pattern, boolean skippable) thr
             if (skippable) {
                 return new Result(true, null);
             }
-            throw new IllegalArgumentException("Key is null or not a string type but pattern contains #{key}");
+            // throw new IllegalArgumentException("Key is null or not a string type but pattern contains #{key}");
```

Comment:
Why is this being commented out now? Is it a bug / should it just be removed?

Reply:
Brought this up in slack and in the earlier PR, when testing with debezium records, found that there were instances with null key, so instead of ignoring, the suggestion was to continue.

Reply:
OK, so we shouldn't throw here? Can we remove the code instead of commenting and describe why in the commit message?  Checking in commented-out code tends to cause confuse further down the line.

Reply:
Removed.